ResearchKit/Common/ORKHelpers.h

```diff
@@ -34,6 +35,37 @@
 #import <ResearchKit/ORKErrors.h>
 
 
+#if ( defined(ORK_LOG_LEVEL_NONE) && ORK_LOG_LEVEL_NONE )
+#  undef ORK_LOG_LEVEL_DEBUG
+#  undef ORK_LOG_LEVEL_WARNING
+#  undef ORK_LOG_LEVEL_ERROR
+#endif
+
+#if ( !defined(ORK_LOG_LEVEL_NONE) && !defined(ORK_LOG_LEVEL_DEBUG) && !defined(ORK_LOG_LEVEL_WARNING) && !defined(ORK_LOG_LEVEL_ERROR) )
+#  define ORK_LOG_LEVEL_WARNING 1
```

Comment:
Should we default `Release` to `ORK_LOG_LEVEL_NONE`? I think it's kind of nice to default `Release` to *warnings*, or at least to *errors*. Other Apple frameworks are quite verbose on the console log regarding potentially dangerous situations.

Current *ResearchKit* warnings are quite sparse, and they can be potentially important for the developer or the user to see.